types/hapi__hapi/index.d.ts

```diff
@@ -257,7 +257,7 @@ export type RequestEventType = 'peek' | 'finish' | 'disconnect';
  * * 'disconnect' - emitted when a request errors or aborts unexpectedly.
  * [See docs](https://github.com/hapijs/hapi/blob/master/API.md#-requestevents)
  */
-export interface RequestEvents extends Podium {
+export class RequestEvents extends Podium {
```

Comment:
This seems incorrect, this being a class implies it would be instantiable eg. be a real type vs. a virtual one, this is not the case, this should stay an interface.

Reply:
You´re right, I corrected it.